fiftyone/core/runs.py

```diff
@@ -241,7 +241,7 @@ def register_run(self, samples, key, overwrite=True):
         run_info = run_info_cls(
             key, version=version, timestamp=timestamp, config=self.config
         )
-        self.save_run_info(samples, run_info)
+        self.save_run_info(samples, run_info, overwrite)
```

Comment:
This change isn't needed. `validate_run()` already handles the `overwrite` parameter